docs/kb/semgrep-appsec-platform/semgrep-login-cli-tenant.md

```diff
@@ -0,0 +1,26 @@
+---
+description: Fix a semgrep login issue for single-tenant customers.
+tags:
+  - Semgrep AppSec Platform
+  - Login
+  - Troubleshooting
+---
+
+
+
+## semgrep login doesn't redirect to my Semgrep site.
+
+When executing the command: 
+```console
+semgrep login
+```
+it redirects to the default site: `https://semgrep.dev` and it could lead to an authentication issue if you are a single-tenant customers, as the expected site should be an URL like: `https://yourcompany.semgrep.dev`
```

Comment:
This line seems unclear to me. It definitely will lead to an issue, and we should be specific about what the issue is (unable to log in/activate token). I think I'd write something like this:

it redirects to [https://semgrep.dev](https://semgrep.dev), and I receive an error {whatever the error is}. I'm expecting to be directed to `https://mycompany.semgrep.dev`.